rust/src/api.rs

```diff
@@ -432,14 +438,20 @@ impl<'a> Endpoint<'a> {
         .await?)
     }
 
-    pub async fn get_stats(&self, app_id: String, endpoint_id: String) -> Result<EndpointStats> {
+    pub async fn get_stats(
+        &self,
+        app_id: String,
+        endpoint_id: String,
+        options: Option<EndpointStatsOptions>,
```

Comment:
This is a breaking change, we should have added a `get_stats_with_options`, though I'm actually fine with this subtle break in Rust land tbh, as the strong version pinning make it mostly a non-issue.